src/features/lookup_table/use_cases/get_lookup_table.py

```diff
@@ -0,0 +1,8 @@
+from authentication.access_control import DEFAULT_ACL, access_control
+from authentication.models import AccessLevel, User
+from storage.internal.lookup_tables import get_lookup
+
+
+def get_lookup_table_use_case(name: str, user: User) -> dict:
```

Comment:
from the name alone I don't really understand what value "name" should have.

in get_lookup() from lookup_tables.py it looks like an uid should be passed in. 
But from the endpoint /application/{application} it looks like an application name is passed in.

Maybe rename the parameter?

Reply:
renamed to lookup_id :+1: